spec/views/groups/new.html.haml_spec.rb

```diff
@@ -0,0 +1,21 @@
+require 'rails_helper'
+
+RSpec.describe "groups/new", type: :view do
```

Comment:
Prefer single-quoted strings when you don't need string interpolation or special symbols.